benchto-service-docker/config/graphite/storage-schemas.conf

```diff
@@ -0,0 +1,10 @@
+# Schema definitions for whisper files. Entries are scanned in order,
+# and first match wins.
+#
+[carbon]
+pattern = ^carbon\.
+retentions = 1s:1d,1min:90d
```

Comment:
Would it be possible to document/explain why there are two definitions (carbon and default) and which is more important?